run-ovms.sh

```diff
@@ -124,7 +117,8 @@ fi
 RUN_MODE="-itd"
 if [ "$RENDER_MODE" == 1 ]
 then
-	xhost +local:docker
+	#xhost +local:docker
+	xhost +local:
```

Comment:
```suggestion
	xhost +local:docker
```

Reply:
is docker user needed? or it can be anyone

Reply:
I haven't tested it in any other way. I know this command works to display UI from a container to the localhost

Reply:
I think this is the safer method. Opening the local to any user could be seen as a security risk.

Reply:
agreed